File: help/pt_pt/exercicios_da_ficha_3/Ficha03_001/Ficha03_001.cpp
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

#include <iostream>
#include <cstring>
#include <cmath>
using namespace std;

// Class definition
class Ficha03_001 {

    public:
        int main() {
            cout << "Base \t\t: ";
            float base;
            cin >> base;
            cout << "Expoente \t: ";
            float expo;
            cin >> expo;
            cout << "\nPotência \t= " <<( pow( base , expo )  );
            cout << "\nOperador \t= " <<( pow( base , expo )  );
            return 0;
        }
};// end of class Ficha03_001
// Execute main function
int main(){
    Ficha03_001 application; // create object
    application.main(); // execute object
    return 0;
}

//C++  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 14:28

File: help/pt_pt/exercicios_da_ficha_3/Ficha03_001/Ficha03_001.php
<?php
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    echo  "Base         : " ;
    $base = floatval(isset($_POST["base"])? $_POST["base"] : isset($_GET["base"]) ? $_GET["base"] : 0);
    
    echo $base . "<br/>";// print input 
    echo  "Expoente     : " ;
    $expo = floatval(isset($_POST["expo"])? $_POST["expo"] : isset($_GET["expo"]) ? $_GET["expo"] : 0);
    
    echo $expo . "<br/>";// print input 
    echo  "\nPotência     = " . ( pow( $base , $expo )  ) ;
    echo  "\nOperador     = " . ( $base ^ $expo  ) ;
}

main();

//PHP 5.3 +  - Automatic translated by Algorithmi 22.05
//(c) Luís Lopes 	Mon, 2022 Sep 26 14:28

?>